force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -191,17 +199,27 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String getDayOfWeek(DateTime dt) {
         // Define a map of day of the week as a key and day number as the value
+        Map<String, Integer> weekDays = new Map<String, Integer>();
         // Monday = 1, Tuesday = 2, etc.
-
+        weekDays.put('Monday', 1);
+        weekDays.put('Tuesday', 2);
+        weekDays.put('Wednesday', 3);
+        weekDays.put('Thursday', 4);
+        weekDays.put('Friday', 5);
+        weekDays.put('Saturday', 6);
+        weekDays.put('Sunday', 7);
         // Get the day name using .format('EEEE');
-
+        String dayEEEE = '';
+        System.debug(dayEEEE);
+        dayEEEE = Datetime.now().format('EEEE');
+        System.debug(dayEEEE);
         // Get the day number from the map using the day name
-
+        String result= String.valueOf(weekDays.get(dayEEEE));
+        System.debug(result);
         // Return the result as a string
-        return null; // Replace null with the variable you used to store the result
+        return result; // Replace null with the variable you used to store the result
```

Comment:
This question is intentionally verbose for practice, but can you do the same in just 1 line using a variation what is already here?